Assets/Content/Furniture/Machines/Science/CellCharger.cs

```diff
@@ -49,10 +54,58 @@ private void Recharge()
             IChargeable chargeable = item.GetComponent<IChargeable>();
             if (chargeable != null)
             {
-		// Effectively add charge
                 chargeable.AddCharge(chargeable.GetChargeRate());
+                UpdateLights(chargeable.GetPowerPercentage());
             }
-            
+        }
+    }
+    
+    /// <summary>
+    /// Turn off all lights
+    /// </summary>
+    private void TurnOffLights() 
+    {
+        redLight.TurnLightOff();
+        orangeLight.TurnLightOff();
+        yellowLight.TurnLightOff();
+        greenLight.TurnLightOff();
```

Comment:
I think it would make a lot of the code shorter if you create an array of all the lights (like `lights = new[] { redLight, orangeLight, yellowLight, greenLight }`.
Then you can just call `TurnLightOff` for each element.

Reply:
Recommendation supported, but left for consideration for future work.